sh/travis.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# change to the top-level bhima directory
+#cd ..
+
+# build the application
+gulp build
+
+# lint the client
+gulp lint
+
+# lint the server
+jshint server/{controllers,lib,config}/*.js
```

Comment:
@jniles I'm guessing it would be best to use gulp uniformly to lint both the client and the server? - this doesn't matter until we have addressed lint issues on the server.